### PR TITLE
Fix/ledger_script

### PR DESCRIPTION
**Pull request by stepanLav**

### CODE REVIEW

**Review by ERussel — APPROVED** [review]

File: chains/v20/chains_dev.json [hunk 1]
```diff
@@ -5114,7 +5118,8 @@
         ],
         "additional": {
             "feeViaRuntimeCall": true,
-            "disabledCheckMetadataHash": true
+            "disabledCheckMetadataHash": true,
```
Comment (ERussel) [inline chains/v20/chains_dev.json:5121]: we need to remove this one if we support Ledger

File: chains/v20/chains_dev.json [hunk 2]
```diff
@@ -2432,7 +2433,8 @@
...
```
Comment (leohar) [inline chains/v20/chains_dev.json:2436]:
```suggestion
```

File: chains/v20/chains_dev.json [hunk 3]
```diff
@@ -5114,7 +5117,8 @@
...
```
Comment (leohar) [inline chains/v20/chains_dev.json:5120]:
```suggestion
```

File: chains/v20/chains_dev.json [hunk 4]
```diff
@@ -9792,7 +9811,8 @@
             "themeColor": "#1F78FF",
             "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/manta-manta-staking",
             "defaultBlockTime": 12000,
-            "disabledCheckMetadataHash": true
+            "disabledCheckMetadataHash": true,
```
Comment (leohar) [inline chains/v20/chains_dev.json:9814]:
```suggestion
```

**Pull request merged**